app/goal_routes.py

```diff
@@ -0,0 +1,97 @@
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
```

Comment:
I like that you separated your routes into two different files to keep your project organized. goal_routes.py and taks_routes.py could also live in a routes directory

Since each route file is for an individual model, we could rename goal_bp to just bp.